app/address/[address]/layout.tsx

```diff
@@ -249,12 +256,12 @@ function AccountHeader({ address, account, tokenInfo, isTokenInfoLoading }: { ad
         let unverified = false;
 
         // Fall back to legacy token list when there is stub metadata (blank uri), updatable by default by the mint authority
-        if (!parsedData?.nftData?.metadata.data.uri && tokenInfo) {
+        if (!parsedData?.nftData?.metadata.uri && tokenInfo) {
```

Comment:
did this recently stop working, or did the `nftData.metadata` payload change?

Not clear to me why this needs to be removed

Reply:
The payload changed. This is using the up to date mpl-token-metadata API with better organization and representation of all fields.

Reply:
Damn rip. Is there easy way to get a test for this in CI, that way I can monitor if API response schema changes?

Reply:
It's not an API response schema. It's the newer SDK versions we launched in 2023. It won't break unless you upgrade, which this PR handles.

Reply:
Sweet